source/includes/legacy-redirect.rst

```diff
@@ -0,0 +1,5 @@
+.. note:: Legacy API
+
+   If you are using the legacy API, 
+   :ref:`see our FAQ page <faq-legacy-connection>`
+   to learn what changes you need to make to this code example.
```

Comment:
```suggestion
   to determine what changes you need to make to this code example.
```
Nit: I prefer the previously worded way because the reader isn't technically learning.

Reply:
I am not sure I agree with this suggestion. Here is my reasoning:

[Learn Definition](https://www.merriam-webster.com/dictionary/learn):  to gain knowledge or understanding of or skill in by study, instruction, or experience

By following the link, I believe readers gain knowledge of constructing a connection with the legacy API by instruction. Therefore I think learn is an appropriate word. In addition, I feel "learn" is more [conversational](https://docs.mongodb.com/meta/style-guide/style/voice-and-tone/#voice-and-tone-attributes) than "determine".